genie-agent/src/main/java/com/netflix/genie/agent/execution/process/impl/JobProcessManagerImpl.java

```diff
@@ -104,11 +102,9 @@ public void launchProcess(
 
         this.initFailedFileRef.set(PathUtils.jobSetupErrorMarkerFilePath(jobDirectory).toFile());
 
-        log.info(
-            "Executing job script: {} (working directory: {})",
+        log.info("Executing job script: {} (working directory: {})",
             jobScript.getAbsolutePath(),
-            launchInJobDirectory ? jobDirectory : Paths.get("").toAbsolutePath().normalize().toString()
-        );
+            launchInJobDirectory ? jobDirectory : Paths.get("").toAbsolutePath().normalize().toString());
```

Comment:
This also seems like an odd formatting change where it pulled the `);` up a line and removed the first break. Generally as a "rule" we've gone with once we need multiple lines for a function call we try to keep one argument per line just for readability and then closing `);` on the subsequent line to delineate blocks